runtime/jcl/common/jclvm.c

```diff
@@ -300,7 +452,76 @@ hasConstructor(J9VMThread *vmThread, J9StackWalkState *state)
 	}
 }
 
+static J9HashTable *
+collectHeapStatistics(J9VMThread *vmThread) {
+	J9JavaVM *vm = vmThread->javaVM;
+	J9HashTable *hashTable = hashTableNew(
+			OMRPORT_FROM_J9PORT(vm->portLibrary),
+			J9_GET_CALLSITE(),
+			64,
```

Comment:
What is `64`?

Reply:
It is the initial size of the table.  I added a comment.